### PR TITLE
Resolved issue #2440  "No images found" 

### PR DESCRIPTION
**Pull request by jainharsh96**
this issue is resolved.
Now bookmarked images are loading in this activity.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,10 +6,10 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:3.3.1'
         classpath 'com.dicedmelon.gradle:jacoco-android:0.1.3'
         classpath 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.8.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.20"
```
Comment (nicolas-raoul) [inline build.gradle]: Did you intend to commit this file too? This seems unrelated, so better put into a different pull request, if useful.

File: app/src/main/java/fr/free/nrw/commons/Media.java [hunk 2]
```diff
@@ -435,21 +435,56 @@ public boolean getRequestedDeletion(){
...
-        if(imageInfo == null) {
+        if (imageInfo == null) {
             return null;
         }
+
+        String imagedescription;
+        try {
+            imagedescription = imageInfo.getMetadata().imageDescription().value();
+        } catch (NullPointerException e) {
+            imagedescription = "null";
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/Media.java]: Return empty values instead. :)

File: app/src/main/java/fr/free/nrw/commons/Media.java [hunk 3]
```diff
@@ -435,21 +435,56 @@ public boolean getRequestedDeletion(){
...
+            dateUploaded = DateUtils.getDateFromString(imageInfo.getMetadata().getDateTime().value());
+        } catch (NullPointerException e) {
+            dateUploaded = new Date();
+        }
+
+        String creator;
+        try {
+            creator = StringUtils.getParsedStringFromHtml(imageInfo.getMetadata().getArtist().value());
+        } catch (NullPointerException e) {
+            creator = "null";
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/Media.java]: Return empty value.

File: app/src/main/java/fr/free/nrw/commons/Media.java [hunk 4]
```diff
@@ -435,21 +435,56 @@ public boolean getRequestedDeletion(){
...
+                dateUploaded,
+                creator
         );
 
-        media.setLicense(imageInfo.getMetadata().getLicenseShortName().value());
+        String license;
+        try {
+            license = imageInfo.getMetadata().getLicenseShortName().value();
+        } catch (NullPointerException e) {
+            license = "null";
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/Media.java]: return empty value

**Comment by maskaravivek** [comment]
Thanks for making the changes @jainharsh96. Can you briefly explain why images were not loading earlier? 

Also, even without this fix, bookmarked images load for me. :/

**Comment by jainharsh96** [comment]
In my case image not loading. same like for the issue was created "No image found".
I checked for the issue and find that there is a problem in Media class 
inside this class 'from' method throwing null pointer exception, so for handling exceptions I made try and cache block for its every arguments.

**Comment by jainharsh96** [comment]
Why is this error arriving? 
while I resolved the issue.

**Comment by domdomegg** [comment]
> Why is this error arriving?
> while I resolved the issue.

It's because of the changes you made in `build.gradle` and `gradle-wrapper.properties`. I'm going to try to undo them now.

**Comment by domdomegg** [comment]
Actually I think the original repository you started this PR from has been deleted. I therefore can't merge this, so am closing. Please reopen a new PR from a public repository if you have a fix!
